router.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"github.com/gorilla/mux"
+)
+
+type RouterEnv struct {
+	service *Service
+}
+
+func (r *RouterEnv) newRouter() *mux.Router {
+	router := mux.NewRouter()
+
+	indexHandlerEnv := HandlersEnv{r.service}
+	indexHandler := indexHandlerEnv.indexHandler
+	devicesHandlerEnv := HandlersEnv{r.service}
```

Comment:
You create two instances of the same handler while you need only one.

Also, maybe `RouterEnv` can be reduced to just `func newRouter(service *Service)`?